.github/workflows/ci.yaml

```diff
@@ -0,0 +1,271 @@
+name: SeqAn3 CI
+
+on:
+  push:
+    branches:
+      # Push events to branches matching refs/heads/master
+      - 'master'
+  pull_request:
+
+env:
+  CMAKE_VERSION: 3.19.1
```

Comment:
In the app template we currently require only CMake version ≥ 3.8.2. 
Should we adapt all CMake files or stay here with 3.8.2? Which features would we need from newer CMake?

Reply:
Changed it to 3.8.2 :)